Source/CNTKv2LibraryDll/proto/onnx/CNTKToONNX.cpp

```diff
@@ -7087,7 +7098,8 @@ void CNTKToONNXHelper::CopyAttributes(const FunctionPtr& src, onnxruntime::Node*
             auto lowerPad = ToINTS(src->Attributes()[L"lowerPad"].Value<NDShape>());
             auto upperPad = ToINTS(src->Attributes()[L"upperPad"].Value<NDShape>());
 
-            if (IsPadValueValid(lowerPad, upperPad, autoPadding, ceilOutDim))
+            // lowerPad and upperPad have incorrect dimension when the op has both batch and sequence axes.
+            if (IsPadValueValid(lowerPad, upperPad, autoPadding, ceilOutDim) && !(src->Inputs()[0].HasBatchAxis() && src->Inputs()[0].HasSequenceAxis()))
```

Comment:
if op has both batch and sequence axes, we consider pad value invalid, call PutPadAttrInNode in the else branch. Is PutPadAttrInNode correct in this scenario?